### DISCUSSION TITLE
Add support tabla and columns Naming Conventions for popular frameworks witch Laravel, Django, Rails

### DISCUSSION THREAD

**Discussion by dialmedu** [Budibase/budibase#12658] [Ideas]
Currently, the naming of tables and columns is quite custom, lacking defined structure. The suggestion here is to introduce a configuration option that enables support for naming conventions used in popular frameworks like Laravel, etc. This extension will provide a more developer-friendly approach, allowing seamless integration with various custom backends.

**Comment by melohagan** [comment]
Well there's no reason you couldn't follow your own naming convention when naming tables and columns? Is this something you would want enforced via validation?
